modules/core/src/main/scala/org/scalasteward/core/forge/ForgeType.scala

```diff
@@ -94,6 +103,8 @@ object ForgeType {
     val publicApiBaseUrl = uri"https://api.github.com"
     val diffs: DiffUriPattern = (from, to) => _ / "compare" / s"$from...$to"
     val files: FileUriPattern = fileName => _ / "blob" / "master" / fileName
+    override def pullRequestHeadFor(fork: Repo, updateBranch: Branch): String =
+      s"${fork.owner}:${updateBranch.name}"
```

Comment:
This is line of code crucial to the fix - using

```
s"${fork.owner}:${updateBranch.name}"
```

which removes the `/${fork.repo}`, rather than as it was previously:

https://github.com/scala-steward-org/scala-steward/blob/9a8f6d3ab4927ea8a07cbd91cc8e1de44090654d/modules/core/src/main/scala/org/scalasteward/core/forge/package.scala#L31